arcgis-rest-api/index.d.ts

```diff
@@ -0,0 +1,221 @@
+// Type definitions for arcgis-to-geojson-utils 10.4
```

Comment:
since these are typings for `arcgis-to-geojson-utils`, wouldn't it be the most helpful to use the same project name here?